### PR TITLE
support `mode` on physical machine chaos

### LINKED ISSUE #2816: PhysicalMachineChaos: spec.mode: Unsupported value: ""
## Bug Report

**What version of Chaos Mesh are you using?**

latest master: ea9c6cc646a17fe0912128f157cb1c4983f4ced7

**What did you do?**

create PhysicalMachineChaos in Dashboard

**What did you expect to see?**

create success

**What did you see instead?**

uWL0YoyUob

It's because it lacks the 'mode' config.

### PR DESCRIPTION
**Pull request by iguoyr**
Signed-off-by: SiyuChen <ryougi201@gmail.com>

### What problem does this PR solve?

Close #2816

### What's changed and how it works?

add `mode` input box on physical machine chaos

### Related changes

- [ ] Need to update `chaos-mesh/website`
- [ ] Need to update `Dashboard UI`
- Need to **cheery-pick to release branches**
  - [ ] release-2.1
  - [ ] release-2.0

### Checklist

Tests

- [ ] Unit test
- [ ] E2E test
- [ ] No code
- [ ] Manual test (add steps below)

Side effects

- [ ] Breaking backward compatibility

### Release note 

```text
Please add a release note.

You can safely ignore this section if you don't think this PR needs a release note.
```

### DCO

If you find the DCO check fails, please run commands like below (Depends on the actual situations. For example, if the failed commit isn't the most recent) to fix it:

```shell
git commit --amend --signoff
git push --force
```

### CODE REVIEW

File: ui/app/src/components/NewExperimentNext/Step1.tsx [hunk 1]
```diff
@@ -60,6 +61,33 @@ const useStyles = makeStyles((theme) => {
 
 const submitDirectly = ['pod-failure']
 
+interface TypeCardProp {
+  name: Env
+  handleSwitchEnv: any
```
Comment (iguoyr) [inline ui/app/src/components/NewExperimentNext/Step1.tsx:66]: @g1eny0ung I have no idea how to fill the type here (so I put `any` for now 😢 ), do you have any specific suggestions?

Comment (g1eny0ung) [reply ui/app/src/components/NewExperimentNext/Step1.tsx:66]: Should be `(env: Env) => () => void`.

Comment (g1eny0ung) [reply ui/app/src/components/NewExperimentNext/Step1.tsx:66]: BTW for this situation, I usually copy from TS IntelliSense in VS Code then change it or not as appropriate. This makes it more convenient:

image

Comment (iguoyr) [reply ui/app/src/components/NewExperimentNext/Step1.tsx]: Got it! Fixed.

File: ui/app/src/components/NewExperimentNext/Step1.tsx [hunk 2]
```diff
@@ -60,6 +61,33 @@ const useStyles = makeStyles((theme) => {
...
+      className={clsx(classes.card, env === name ? classes.cardActive : '')}
+      variant="outlined"
+      onClick={handleSwitchEnv(name)}
+    >
+      <Box display="flex" justifyContent="center" alignItems="center" width={225} height={75}>
+        <Box display="flex" justifyContent="center" flex={1}>
+          {iconByKind(name)}
+        </Box>
+        <Box flex={1.5} textAlign="center">
+          <Typography variant="button">{T(`${title}`)}</Typography>
```
Comment (g1eny0ung) [inline ui/app/src/components/NewExperimentNext/Step1.tsx:84]: T(`${title}`) is equal with `T(title)`. It's better to pass the `title` directly.

**Review by g1eny0ung — APPROVED** [review]
Rest LGTM 👍🏽

File: ui/app/src/components/NewExperimentNext/Step1.tsx (hunk 2 above)
Comment (iguoyr) [reply ui/app/src/components/NewExperimentNext/Step1.tsx]: Fixed.

**Review by WangXiangUSTC — APPROVED** [review]
LGTM

**Pull request merged**